scripts/import/dbSNP_v2/load_dbsnp.pl

```diff
@@ -1587,6 +1616,16 @@ sub get_seq_region_names {
   return (\%seq_region);
 }
 
+# For a given chr look up the seq_region
+sub get_seq_region_chr {
+  my ($dbh, $chr) = @_;
```

Comment:
This relies on only one seq_region record for Y being imported from core to variation. We should probably check somewhere.

Reply:
A comment has been added and a ticket created to check this at the start of the script.